pages/task/read/routers/read.js

```diff
@@ -11,7 +11,8 @@ const { getEstablishment } = require('../../../common/helpers');
 const updateData = require('../middleware/update-data');
 
 const endorsingOwnPil = (task, profile) => {
-  return task.data.model === 'pil' && task.status === 'awaiting-endorsement' && profile.id === get(task, 'data.subject.id');
+  const isNtco = !!profile.roles.find(r => r.type === 'ntco' && r.establishmentId === task.data.establishmentId);
+  return task.data.model === 'pil' && task.status === 'awaiting-endorsement' && isNtco && profile.id === get(task, 'data.subject.id');
```

Comment:
```suggestion
  return isNtco && task.data.model === 'pil' && task.status === 'awaiting-endorsement' && profile.id === get(task, 'data.subject.id');
```

I don't know why, but having that condition right in the middle bothered me.